### PR TITLE
[MM-48391] Add export version info to the docs

### PR DESCRIPTION
**Pull request by noxer**
#### Summary
Export files will include additional version info in the future. This PR adds the changes to the docs.

#### Ticket Link
[MM-48391](https://mattermost.atlassian.net/browse/MM-48391)

#### Server PR
https://github.com/mattermost/mattermost-server/pull/21709

### CODE REVIEW

**Review by cwarnermm — APPROVED** [review]
Thanks, @noxer!

File: source/manage/bulk-export-data.rst [hunk 1]
```diff
@@ -75,8 +75,50 @@ Version object
...
+    <tr class="row-odd">
+      <td valign="middle">version</td>
+      <td valign="middle">string</td>
+      <td>The version of the tool this export was generated with. This may contain multiple pieces of version info, separated by spaces. The first one should be a semantic version.<br>
+      <kbd>"7.6.0 (29bb1e53ef5a439c73065f47de2972f9bbcb09a4, enterprise: true)"</kbd> is an example of such a version string.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">created</td>
+      <td valign="middle">string</td>
+      <td>The timestamp of the file creation. This should be formatted as a RFC 3339 timestamp. The nanosecond part is optional.<br>
```
Comment (agarciamontoro) [inline source/manage/bulk-export-data.rst:112]:
```suggestion
      <td>The timestamp of the file creation. This should be formatted as an RFC 3339 timestamp. The nanosecond part is optional.<br>
```

**Review by agarciamontoro — CHANGES_REQUESTED** [review]
Cool, thanks!! Just a small typo:

**Review by agarciamontoro — APPROVED** [review]
:tada:

**Pull request merged**